test_player.py

```diff
@@ -91,7 +91,7 @@ def test_quit_command(self):
         valid_parameters = VALID_ARGS()[1]
 
         with player_context(valid_parameters, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL) as program:
-            time.sleep(PLAYER_BOOT_SECONDS) # lepiej poczekac, bo dziwne akcje odwala...
+            time.sleep(PLAYER_BOOT_SECONDS)
```

Comment:
Przerzucisz to proszę do `player_context`? Tak będzie prościej, bo i tak wszędzie to mamy.